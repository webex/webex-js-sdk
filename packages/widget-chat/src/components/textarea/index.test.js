import React from 'react';
import renderer from 'react-test-renderer';

import TextArea from '.';

describe(`TextArea component`, () => {
  const props = {
    className: `test-class`,
    onChange: jest.fn(),
    onKeyDown: jest.fn(),
    onSubmit: jest.fn(),
    placeholder: `TextArea Placeholder`,
    rows: 2,
    value: ``
  };

  const component = renderer.create(
    <TextArea {...props} />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
