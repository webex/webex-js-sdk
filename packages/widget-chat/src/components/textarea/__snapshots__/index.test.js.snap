exports[`TextArea component renders properly 1`] = `
<textarea
  className="textarea"
  onChange={[Function]}
  onKeyDown={[Function]}
  onSubmit={[Function]}
  placeholder="TextArea Placeholder"
  rows={2}
  value="" />
`;
